resources/views/admin/adminUpdate.blade.php

```diff
@@ -1,6 +1,29 @@
 @extends('layouts.master')
 
 @section('body')
+<script src="https://code.jquery.com/jquery-1.11.1.min.js" ></script>
+<script src="https://cdn.jsdelivr.net/gh/rexeze/formatTimeStamp/src/index-cdn.js" ></script>
```

Comment:
Do we need these two lines? JQuery should be included by default with Bootstrap (included in master.blade.php). Not sure what the second library is doing...